Tests.HealthCheck/ReportEmailSender.cs

```diff
@@ -65,7 +68,7 @@ public static void SendEmails()
                 }
                 catch (Exception )
                 {
-                    
```

Comment:
Why Exception is empty?

Reply:
It is not my code and my task, I only have to remove credentials